### ISSUE TITLE
<script> sections not being called when component is late rendered

### ISSUE THREAD

**Issue by martindemello** [adamghill/django-unicorn#436]
I have several components of the form
```
  <div>
    {{ round_pairing | json_script:"round-pairings-data" }}
    <div unicorn:ignore id="tabulator-table"></div>
    <script src="{% static "tournament/round_pairings.js" %}"></script>
  </div>
```
which load data for a table and then call a javascript function to populate it in the `tabulator-table` div. The javascript file is different for each component, and calls `makeTable()` at top level at the end.

This works fine when I include a component directly in a page. Now I'm trying to write a tab view, with the parent looking like
```
      <div>
          <a href="{% url 'tournament:round_pairings' tournament.id div.id %}"
            unicorn:click.prevent="set_active({{div.id}}, 'round-pairings')"
          >Round Pairings</a>
          {% if active_view %}
          {% unicorn active_view division=active_division parent=view key=active_key %}
          {% endif %}
      </div>
```
where `active_view` is the name of the child component, and it looks like the javascript is never getting loaded, and furthermore that any script tags in the child component are not getting executed.

I suspect this is the same issue as https://github.com/adamghill/django-unicorn/pull/243 and that there's no good solution, but filing this just in case there's been a fix or some better way of doing things.

**Comment by martindemello** [comment]
thanks! i'm still getting to grips with the code but i'll give it a try and see if i can get anywhere. (i also filed #437 in the process of trying to find a different way to do things)

**Comment by adamghill** [comment]
https://github.com/adamghill/django-unicorn/pull/442 is one approach which will reload the script tag after a template has been morphed. I am going to hide it behind a setting for now, though.

**Comment by adamghill** [comment]
I have added a fix for this in 0.48.0. You can enable it with the [`RELOAD_SCRIPT_ELEMENTS` setting](https://www.django-unicorn.com/docs/settings/#reload-script-elements).
